lib/netext/grpcext/stream.go

```diff
@@ -68,8 +68,8 @@ func (s *Stream) receive() (*dynamicpb.Message, error) {
 //
 //	message Point {
 //	   double x = 1;
-//		  double y = 2;
-//		  double z = 3;
+//		double y = 2;
+//		double z = 3;
```

Comment:
nit: not certain what happened here, but I can't propose a change through the UI that seems right :(